src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,53 @@
 import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard:React.FC<Props> = ({ movie }) => {
+  return (
```

Comment:
```suggestion
export const MovieCard:React.FC<Props> = ({ movie }) => (
```

You can not use `return` in arrow function